photon-server/src/main/resources/web/index.html

```diff
@@ -1 +1,14 @@
-<p>UI has not been copied!</p>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Photon Client</title>
+    <script type="module" crossorigin src="./assets/index-C3iBAvTn.js"></script>
+    <link rel="stylesheet" crossorigin href="./assets/index-BPEyX_3L.css">
+  </head>
+  <body>
+    <div id="app"></div>

+  </body>
+</html>

```

Comment:
This file should not be committed. It's in .gitignore so that it isn't changed from the default message. 
```suggestion
<p>UI has not been copied!</p>
```